tests/post_training/test_quantize_conformance.py

```diff
@@ -174,6 +178,9 @@ def fixture_ptq_report_data(output_dir, run_benchmark_app, pytestconfig):
         if not run_benchmark_app:
             df = df.drop(columns=["FPS"])
 
+        df = df.drop(columns=["Num sparse activations"])
```

Comment:
looks like report_dat can be use one function to generate csv,
some thing like `save_results(daf, droped_columns)`